src/components/ui/header.tsx

```
@@ -0,0 +1,36 @@
+import Image from 'next/image';
+
+interface HeaderProps {
+  onBack: VoidFunction;
+  onClose?: VoidFunction;
+}
+
+const Header = ({ onBack, onClose }: HeaderProps) => {
+  return (
+    <>
+      <header className='absolute top-0 left-0 flex h-11 w-full items-center justify-between px-4'>
+        <button onClick={onBack}>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)